compiler/code-gen/declarations.cpp

```diff
@@ -949,10 +945,6 @@ void ClassDeclaration::compile_accept_visitor_methods(CodeGenerator &W, ClassPtr
 }
 
 void ClassDeclaration::compile_msgpack_declarations(CodeGenerator &W, ClassPtr klass) {
-  if (G->is_output_mode_k2()) {
```

Comment:
Why you remove this condition without adding definition of `vk::msgpack::packer` for runtime light? Compilation of classes with `/** @kphp-serializable */` tag fails

Reply:
+